### PR TITLE
Add anti-bribery and events policies

### PR DESCRIPTION
**Pull request by vollmera**

### CODE REVIEW

**Review by mlinksva — APPROVED** [review]
Exact copy of additional policies now published at https://help.github.com/categories/site-policy/

**Comment by KOLANICH** [comment]
I wonder if this a result of MS acquisition or they have been existing long before, but were just not published.

**Comment by insanm61** [comment]
Ok

File: Policies/github-anti-bribery-statement.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
+---
+title: GitHub Anti-Bribery Statement
```
Comment (luismedicina23) [inline Policies/github-anti-bribery-statement.md:2]: @Autor

**Review by Budda0ne — COMMENTED** [review]
Рев


```

**Comment by d066** [comment]
thanks

**Comment by codedfile** [comment]
https://github.com/Hacker0x01/docs.hackerone.com/commit/b3c0e0f01c6d45066be08635c83d09aca033487c#commitcomment-54728323

**Review by Hecim1984 — COMMENTED** [review]
🎉🚀

**Comment by 966563058015** [comment]
image

**Review by NikkieDickie — COMMENTED** [review]
> 
411

**Review by Jelisacooper4 — COMMENTED** [review]
#DoubleGrowthLLC

**Review by ghost — COMMENTED** [review]
Q

**Review by Neo-wise — APPROVED** [review]

**Comment by ALLARDLEE** [comment]
Support

**Comment by ALLARDLEE** [comment]
Support

**Comment by urHeQWjQ2D6GGLz** [comment]
>

**Comment by urHeQWjQ2D6GGLz** [comment]
- [ ] >

**Review by Neo-wise — APPROVED** [review]

**Review by Systemcyruss88 — APPROVED** [review]
Policies/github-anti-bribery-statement.md/cryptoversion

File: Policies/github-event-terms.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
+---
+title: GitHub Event Terms
```
Comment (Bezzo100-lab) [inline Policies/github-event-terms.md:2]: Amanda Brandenburgh

**Review by Bezzo100-lab — APPROVED** [review]

**Comment by ziggy74ai** [comment]
What does this mean

File: Policies/github-gifts-and-entertainment-policy.md [hunk 3]
```diff
@@ -0,0 +1,60 @@
+---
+title: GitHub Gifts and Entertainment Policy
```
[the remark this answers is not in the document]
Comment (Neowise01) [reply Policies/github-gifts-and-entertainment-policy.md:2]: > Policies/github-event-terms.md

File: Policies/github-event-code-of-conduct.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+#### Reporting an incident
+
+If you are being harassed, notice that someone else is being harassed, or have any other concerns, please contact the organizer by email (events@github.com) or phone/text (+1 415 842 2837), or by speaking to another GitHub employee or a security officer.
+
+GitHubbers will be happy to help participants contact security or local law enforcement, escort to safety, or otherwise assist those experiencing harassment to feel safe for the duration of the conference. We value your attendance.
+
+### Credit
+
+Portions of this Code of Conduct are based on the [example anti-harassment policy](http://geekfeminism.wikia.com/wiki/Conference_anti-harassment/Policy) from the Geek Feminism wiki, created by the Ada Initiative and other volunteers, under a Creative Commons Zero license.
```
Comment (ziggy74ai) [inline Policies/github-event-code-of-conduct.md:44]: 🙃[]()
WZ00️⃣😔😔🎊0️⃣0️⃣

File: Policies/github-anti-bribery-statement.md [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+---
+GitHub upholds our commitment to ethical business practices, including by prohibiting bribery and corruption. As [Transparency International](https://www.transparency.org/what-is-corruption#costs-of-corruption) explains, the costs of corruption are
```
Comment (ziggy74ai) [inline Policies/github-anti-bribery-statement.md:4]: https://t.me/+lllloex44848dd9r440

**Review by ziggy74ai — APPROVED** [review]
> > > > Kt09Sakhal899`~~~~`

File: Policies/github-gifts-and-entertainment-policy.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Chixonya22) [reply Policies/github-gifts-and-entertainment-policy.md:2]: Bit

**Comment by Hwycjcd** [comment]
https://youtube.com/@user-sy3pu6ed5o?si=_MYP_dEqM1isZBj3

File: Policies/github-gifts-and-entertainment-policy.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (Hwycjcd) [reply Policies/github-gifts-and-entertainment-policy.md:2]: https://github.com/github/site-policy/pull/121#pullrequestreview-1488135158

**Comment by 14126120941512121180414934817** [comment]
16181510091614416110318151840966101510185514415611181261518

**Comment by 14126120941512121180414934817** [comment]
****

**Review by Dramaga11 — APPROVED** [review]

**Comment by An010299** [comment]
A educação está evoluindo e com isso o desenvolvimento também.  Aquele que sabe estará  liberto e com maior potencial e inspiração!

File: Policies/github-anti-bribery-statement.md (hunk 5 above)
Comment (An010299) [reply Policies/github-anti-bribery-statement.md:4]: Auditandobe andando criaremos novas canções.

**Review by An010299 — CHANGES_REQUESTED** [review]

**Review by Green369-1 — APPROVED** [review]

**Review by Green369-1 — APPROVED** [review]

File: Policies/github-event-code-of-conduct.md [hunk 6]
```diff
@@ -0,0 +1,44 @@
+---
```
[the remark this answers is not in the document]
Comment (Lobotwallker) [reply Policies/github-event-code-of-conduct.md:1]: Como instalar

**Review by Dixie5644 — APPROVED** [review]
300

**Review by nelegalrap — COMMENTED** [review]
https://github.com/melegalofficial?tab=stars

**Comment by leroy182** [comment]
>

**Review by leroy182 — COMMENTED** [review]
@leroy182

File: Policies/github-anti-bribery-statement.md (hunk 5 above)
Comment (leroy182) [inline Policies/github-anti-bribery-statement.md:4]: @leroy182 to ethical business practices. Including as transparency International. 
https://www.transparency.org/@leroy182

**Review by Phamduy1247 — COMMENTED** [review]
Lo

**Review by Gitrris — APPROVED** [review]

**Review by Gitrris — APPROVED** [review]

**Review by Fiyaas — APPROVED** [review]

**Review by Fiyaas — APPROVED** [review]

**Review by Dustin4444 — APPROVED** [review]

**Review by draghtnight — COMMENTED** [review]
body, html {
    margin: 0;
    padding: 0;
    box-sizing: border-box;
    font-family: Arial, sans-serif;
}

.container {
    max-width: 400px;
    margin: 50px auto;
    padding: 20px;
    border: 1px solid #ddd;
    border-radius: 5px;
    box-shadow: 0 0 10px rgba(0, 0, 0, 0.1);
}

h1 {
    text-align: center;
}

form {
    display: flex;
    justify-content: space-between;
}

input {
    flex-grow: 1;
    padding: 10px;
    border: 1px solid #ddd;
    border-radius: 5px 0 0 5px;
}

button {
    padding: 10px 20px;
    border: none;
    background-color: #28a745;
    color: white;
    cursor: pointer;
    border-radius: 0 5px 5px 0;
}

button:hover {
    background-color: #218838;
}

ul {
    list-style-type: none;
    padding: 0;
}

li {
    display: flex;
    justify-content: space-between;
    padding: 10px;
    border-bottom: 1px solid #ddd;
}

li:last-child {
    border-bottom: none;
}

li button {
    background-color: #dc3545;
}

li button:hover {
    background-color: #c82333;
}

**Review by xupne2-gehcyF-vipdin — COMMENTED** [review]
楽しい

**Comment by shinedura** [comment]
"Glass that glows, aluminium that lasts — crafted by ShineDura hands."

**Comment by otbotb371-lgtm** [comment]
> "Glass that glows, aluminium that lasts — crafted by ShineDura hands."

**Review by dameonx69washington-beep — APPROVED** [review]

**Review by autumnthothincorperated — COMMENTED** [review]
work-horse-re-write-256

**Review by amartinezjr0310310-cyber — COMMENTED** [review]
This is amazing 😀

**Review by 1sttatech-789 — APPROVED** [review]

**Review by grewaldetective7-tech — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Comment by noorizanbintiabdulmalik85** [comment]
Picing and aplications from group companies github interprise 2023,TNB METER WIFI BROKEN,AND DATA

**Pull request merged**